AppCenter/AppCenter/MSAppCenter.m

```diff
@@ -251,7 +252,7 @@ - (void)start:(NSString *)appSecret withServices:(NSArray<Class> *)services from
                    (fromApplication ? @"an application" : @"a library"));
       NSMutableArray<NSString *> *servicesNames = [NSMutableArray arrayWithCapacity:sortedServices.count];
       for (Class service in sortedServices) {
-        if ([self startService:service withAppSecret:appSecret andSendLog:NO fromApplication:fromApplication]) {
+        if ([self startService:service withAppSecret:self.appSecret andSendLog:NO fromApplication:fromApplication]) {
```

Comment:
great catch! I'm thinking maybe we should move `self.appSecret = appSecret` outside of `configureWithSecretString:fromApplication` or at least make it more obvious where we are assigning the value to the property 🤔

Reply:
Yeah I don't know if I would have found this bug without @clpolet's sharp eyes - I'll look at making it clearer.

Reply:
This change was intentional. I needed to use the current appSecret to start service because of some reasons. I'll take a look at this code again.